core/emulate.c

```diff
@@ -765,6 +783,38 @@ static em_status_t decode_op_si(em_context_t *ctxt,
 }
 
 /* Soft-emulation */
+static void em_andn_soft(struct em_context_t *ctxt)
+{
+    uint64_t temp;
+    uint64_t signbit = 63;
+    temp = ~ctxt->src1.value & ctxt->src2.value;
+    if (ctxt->operand_size == 4) {
+        temp &= 0xFFFFFFFF;
+        signbit = 31;
+    }
+    ctxt->dst.value = temp;
+    ctxt->rflags = 0;
+    ctxt->rflags |= (temp == 0) ? RFLAGS_ZF : 0;
+    ctxt->rflags |= (temp >> signbit) ? RFLAGS_SF : 0;
+}
+
+static void em_bextr_soft(struct em_context_t *ctxt)
+{
+    uint8_t start, len;
+    uint64_t temp;
+
+    start = ctxt->src2.value & 0xFF;
+    len = (ctxt->src2.value >> 8) & 0xFF;
+    temp = ctxt->src1.value;
+    if (ctxt->operand_size == 4) {
+        temp &= 0xFFFFFFFF;
+    }
+    temp >>= start;
+    temp &= ~(~0ULL << len);
+    ctxt->dst.value = temp;
+    ctxt->rflags = (temp == 0) ? RFLAGS_ZF : 0;
```

Comment:
Same here. Although the SDM says "All other flags are cleared", I think it means all _status_ flags.